.github/workflows/pr.yml

```diff
@@ -0,0 +1,20 @@
+name: pr
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.14.0"
+      - run: yarn install
+      - run: yarn test
+      - uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Note: the `GITHUB_TOKEN` is built into Github Actions and does not need to be manually specified in your secrets store. 

See [Coveralls GitHub Action](https://github.com/marketplace/actions/coveralls-github-action#inputs) or [Github doc](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret).